composer.json

```diff
@@ -68,7 +68,8 @@
         "illuminate/support": "self.version",
         "illuminate/translation": "self.version",
         "illuminate/validation": "self.version",
-        "illuminate/view": "self.version"
+        "illuminate/view": "self.version",
+        "tightenco/collect": "self.version"
```

Comment:
shouldn't it just be `"tightenco/collect": "*"`. Since any version of `tightenco/collect` should be ignored